Assets/Tests/PlayModeTests/MigrationTests.cs

```diff
@@ -0,0 +1,477 @@
+﻿using System.Collections.Generic;
+using NUnit.Framework;
+using UnityEngine;
+using Plugins.CountlySDK.Models;
+using Plugins.CountlySDK;
+using Newtonsoft.Json;
+using System.Web;
+using System.Collections.Specialized;
+using Newtonsoft.Json.Linq;
+using Plugins.CountlySDK.Helpers;
+using UnityEditor.PackageManager.Requests;
+using System.Linq;
+using iBoxDB.LocalServer;
+using Plugins.CountlySDK.Persistance.Entities;
+using Plugins.CountlySDK.Persistance.Repositories;
+using Plugins.iBoxDB;
+
+namespace Tests
+{
+    public class MigrationTests
+    {
+        private readonly string _serverUrl = "https://xyz.com/";
+        private readonly string _appKey = "772c091355076ead703f987fee94490";
+
+        /// <summary>
+        /// It validates the request migration on empty request repo.
+        /// </summary>
+        [Test]
+        public void TestMigrationOnEmptyRequestRepo()
+        {
+            CountlyConfiguration configuration = new CountlyConfiguration {

```

Comment:
The initial schema version in this test should be 1. Currently it would already be 2 and no migration on empty DB would be run.